### ISSUE TITLE
Milestone 13 feedback

### ISSUE THREAD

**Issue by kandarpksk** [SpiritRushAhri/team17#10]

**Comment by lmgong** [comment]
Team K5: SoDoge
Thomas: The most important thing to finish by next week is your ability to move the circle and broadcast accurately to your other users. This would be useful for people who’re wandering around and trying to either advertise their own music or find other people’s music. It would be great if you can increase your broadcasting range which will have more pins pop up in the region which will allow for a more improved version of your already in place functionality.
 
Lauren: The most important thing to finish by next week is to make sure that the data that you are pulling for the most popular songs in the area is accurate. This would be really useful for helping users find new music that they might be interested in listening to. Additionally, it would be great if you are able to have the pins pop up in the region defined by the user. This allows more flexibility and will help people get a more broad view of what other people near them are listening to. Overall I think it’s a really interesting idea, can’t wait to see how it all comes together!
 
Brian: The most important thing to finish by next week is to finish the circle implementation for your app. Right now, the markers within the central circle are all hard-coded in and are inflexible. By next week, you should finish the implementation that allows the users to change the circle size/radius and to see the markers that are no longer encapsulated by the circle to disappear. Also, ensure that the music data that is displayed is accurate because it’s the most important part of your app.
 
Hyungyun: The most important thing to finish by next week is to finish main feature of your app. I think most important feature is to display rank of songs. I think your team also need to finish circle function. For example, after expand the circle, data result should be reflect based on the circle size. I think that’s relate to the main feature. Also, your app could be improved the UI to be more “music friendly design”.

**Comment by bbqplox** [comment]
JOANNA (K3)

Joseph:
The most important thing to finish by next week is to polish the UI and do more with the data visualization at the bottom of the page. Add color the the app and make it so that the user can update the location and generate new nearby pins with other listeners with last fm data.

Anish:
The most important thing to finish by next week is to maybe make it seem more interactive. Not by functionality as the the product seems pretty complete and basically almost finished. Instead, by adding some color that can change as the user interacts differently with the application to mean different actions. This implementation is completely up to you. Otherwise nice job, it came out pretty well. 

Ashkon:
The the most important thing to finish by next week is adding more polish to the UI. Right now it’s a little hard to follow due to its a lack of color like Nic said. I think another really important feature would to add an actual userbase system. It could be done quickly with something like firebase and then linking with their lastfm account. 

Nic: The most important thing to finish by next week is to perhaps add some color into your application. It’s good that you all started designing in greyscale (like they said to in cogs 120), but some color could add a useful hierarchy to the design. Other than that, the functionality looks complete (to the scope of this class), and the UI looks very intuitive. Perhaps the landing page could use the most work. For example, it wasn’t clear at first that we were supposed to put our lastfm account information.
